### ISSUE TITLE
Final Report review by Zhiting(Vincent)

### ISSUE THREAD

**Issue by vincent6606** [ORIE4741/projects#81]
I enjoyed reading the report as it is concisely written. The graph showing days before readmission versus days spent in the hospital shows surprising information, since I had the assumption that if patients stay in the hospital longer, it is less likely for them to be readmitted. However, the this figure shows the exact opposite; patients who stayed in hospitals longer have more serious conditions, and are therefore more likely to be readmitted. I also liked the way your team showed how the 30-day cut-off is chosen. Figure 2 shows a good justification for this cut-off, which is a good use of exploratory data analysis before applying any prediction techniques. Moreover, I think it is a good way to find a good model by applying different regressions methods paired with multiple regularizes, although I would imagine it would take a fair amount of time. I also like the use of Low-Rank Models as you incorporate a different dataset into your Logistic Classification. I am curious, though, what you found out about your data with the archetypes, as they usually tell you about how each row of data is different from another, i.e. how one patient is different from another.
As for suggestions, I have a hard time understanding the later parts of the report. For example, I do not understand the neural network part of the report. Maybe you could give a brief explanation what neural nets are and how they are relevant to the analysis. Perhaps it would take a significant amount of space to explain neural nets to lay persons, but your report would benefit a lot if readers can clearly tell how the use of the technique would improve the parameters of your regression. 
In conclusion, you report is well written, and is supported by appropriate graphs and numbers. I like the way you searched through different regularizes and models, and the use of GLRM since my team also tried using it. Good job guys!
